foundry.toml

```
@@ -0,0 +1,6 @@
+[profile.default]
+src = 'src'
+out = 'out'
+libs = ['lib']
+
+# See more config options https://github.com/foundry-rs/foundry/tree/master/config
```

```suggestion
[profile.default]
src = 'src'
out = 'out'
libs = ['lib']
```

Maybe we also want to copy more defaults from https://github.com/centrifuge/connectors/blob/main/foundry.toml?